docusaurus/docs/dev-docs/plugins/documentation.md

```diff
@@ -51,332 +52,212 @@ This plugin comes with an interface that is available in your administration pan
 
 By default, your documentation will be accessible by anyone.
 
-If you want to restrict the access to the API documentation you have to enable the **Restricted Access** option.
+If you want to restrict the access to the API documentation you have to enable the **Restricted Access** option.
 
-- Click the `ON` of **Restricted Access**
-- Select a password in the `password` input
+- Go to **Settings**, **Documentation**
+- Toggle **Restricted Access** to `ON`
+- Select a password in the `password` input
 - Save your settings
 
-The API documentation is now password-restricted.
-
-### Retrieve your JWT token
-
-Strapi is secured by default, which means that most of your end-points require the user to be authorized. You need to paste the JWT token into your SWAGGER UI to test your end-points.
-
-1. Click on the **Retrieve your jwt token** input to copy the token.
-2. Visit your documentation.
-3. Click on the **Authorize** button on the right.
-4. Paste your token in the `value` input.
+The API documentation is now password restricted.
 
 ### Regenerate documentation
 
-Documentation is not updated automatically. Whenyou update your API you need to click on the **Regenerate** button for the documentation version you want to update. It regenerates to the specified version with the current API documentation.
-
-## Settings
+Documentation is not updated automatically. When you update your API you need to click on the **Regenerate** button for the documentation version you want to update. It regenerates to the specified version with the current API documentation.
 
-You need to create the `./extensions/documentation/config/settings.json` file manually to customize the swagger ui settings.
+## Configuration
 
-Here is the file contents that need to be created in order to change the documentation version, the server URL and so on.
+The documentation plugin is initialized with the following config and all the properties found here can be altered by providing new values on the documentation plugin's config object in `config/plugins.js`.
 
-```
+```jsx
```

Comment:
don't you mean js? And it would be nice to show the full file, I'm not sure what I need to do with the object when I see this